markets/aave/reservesConfigs.ts

```diff
@@ -264,3 +264,15 @@ export const strategyXSUSHI: IReserveParams = {
   aTokenImpl: eContractid.AToken,
   reserveFactor: '3500',
 };
+
+export const strategyENS: IReserveParams = {
+  strategy: rateStrategyVolatileFour,
+  baseLTVAsCollateral: '5000',
+  liquidationThreshold: '6000',
+  liquidationBonus: '8000',
```

Comment:
Liq bonus should be above 100%.  A liq bonus of 108% means the liquidator will receive a 8% bonus. This value should be written with 2 zeros, so `10800`